packages/core/template_project/app/build.gradle

```diff
@@ -18,9 +18,6 @@ import groovy.xml.MarkupBuilder
 
 plugins {
     id 'com.android.application'
-    <% if (serviceAccountJsonFile) { %>
```

Comment:
Does somehow change that gradle is signing the APKs (opposed to use apksigner?). Asking because I remember this was changed when we introduced this.

Reply:
This change needed our gradle version to be updated, so [I updated it here](https://github.com/GoogleChromeLabs/bubblewrap/pull/491/files#diff-86b7a2a741060fd97a8c701a201b0db24189f86358593672cf3e02d33d0edd15R26). Moving forward, gradle will still do signing of the APKs and apksigner can still check for alignment I believe.